.github/workflows/release.yml

```diff
@@ -284,12 +299,26 @@ jobs:
         if: matrix.config.os == 'windows-latest'
         run: |
           $archive = "jormungandr-${{ needs.create_release.outputs.version }}-${{ matrix.config.target }}-${{ matrix.target_cpu }}.zip"
+
+          (Get-FileHash -Path "./target/${{ matrix.config.target }}/release/jormungandr.exe" -Algorithm SHA256).Hash.toLower() | Out-File -FilePath "./jormungandr.exe.sha256"
+          (Get-FileHash -Path "./target/${{ matrix.config.target }}/release/jcli.exe" -Algorithm SHA256).Hash.toLower() | Out-File -FilePath "./jcli.exe.sha256"
+
           $args = @{
             Path  = "./target/${{ matrix.config.target }}/release/jormungandr.exe",
-                    "./target/${{ matrix.config.target }}/release/jcli.exe"
+                    "./jormungandr.exe.sha256",
```

Comment:
I don't think it's worth putting these hash files into the archive, as you hash the whole archive and publish the checksum file as well, that covers the need for an integrity check. Just write them into `${{ runner.temp }}` for the sanity check below.